compiler/include/ForallStmt.h

```diff
@@ -30,12 +30,12 @@
 class ForallIntent : public Expr 
```

Comment:
Why is there still a class ForallIntent? I thought you were replacing it with ShadowVarSymbol?

Reply:
This is correct. This class should not be used. I would like to remove it in a separate "trivial" PR. OK?

Reply:
If it's trivial enough, why not remove it now? I don't see any advantage to putting it off, other than confusing me :) That is to say, I'd like you to remove it now if it's trivial. If it's not trivial, then you'll learn something, right?

If you want to do this in multiple PRs, the thing to split out would be adding the ShadowVarSymbol - just add it as a class without using it anywhere...